base_comment_template/migrations/15.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    for item in env["base.comment.template"].sudo().search([]):
+        item.models = ",".join(item.mapped("model_ids.model"))
```

Comment:
Doing this, it will trigger again the recomputation of the `model_ids` computed field, so better to write it through SQL.